addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -619,11 +637,12 @@ def test_in_invoice_line_onchange_accounting_fields_1(self):
             },
             {
                 **self.tax_line_vals_2,
-                'price_unit': 250.0,
-                'price_subtotal': 250.0,
-                'price_total': 250.0,
-                'amount_currency': 250.0,
-                'debit': 250.0,
+                'price_unit': -75.0,
+                'price_subtotal': -75.0,
+                'price_total': -75.0,
+                'amount_currency': -75.0,
+                'credit': 75.0,
+                'debit': 0.0,
```

Comment:
Seeing this, I realized that it's not just a small "refactoring" but a whole change of behavior which will have different results. This means that it may be refused.

Reply:
Yes I realize it.

But the current behavior is not correct as a negative line for tax should be generated with the good tag (negative one).

You can reproduce on runbot:

Load a localization with positive and negative tags (l10n_lu for instance).
Create an OD with a sale line (tax line automatically generated, + tag on revenue account), a rebate line (tax line automatically generated, - tag on the other revenue account).

Create an Invoice, with both lines. Check the generated entries.
Only one line has been generated for tax (with the + tag).

Reply:
IMHO, the OD has the correct behaviour.